### PR TITLE
Pill bottles for all

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
The maintainers are furiously throwing dice at each other and we rolled double six's. According to the wheel that means you get pill bottles back.

Replaced packets entirely with pill bottles for marine use.
## Why It's Good For The Game
Unga gets to gobble more pills.
## Changelog
:cl:
balance: Replaced packets with pill bottles for all marine use
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
The great bottling

**Review by Xander3359 — APPROVED** [review]
death to paracematol

File: code/game/objects/items/storage/pill_packets.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@
 	pip_color = COLOR_PACKET_DYLOVENE
 
 /obj/item/storage/pill_bottle/packet/paracetamol
-	name = "paracematol pill packet"
+	name = "paracetamol pill packet"
```
Comment (spookydonut) [inline code/game/objects/items/storage/pill_packets.dm:68]: unatomic change

**Comment by Oscari-pog** [comment]
I know this seems like gimping self but could you leave packets existing

Like put pill bottles on top if you want to, just make packets still available

**Comment by Al-1ce** [comment]
> I know this seems like gimping self but could you leave packets existing


..Why? The only issue I can see is this might bug up some load-outs, but otherwise the packets are completely moot when compared to any of the pill bottles.

**Comment by InterroLouis** [comment]
Can we keep pill packets around for those marines who want to have quick access to pills instead of a 2.5 second delay for every pill they want to pull?

**Comment by InterroLouis** [comment]
Never mind I just missed the fumble revert.

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**